deploy/olm-catalog/qdrouterd-operator/0.1.0/qdrouterd-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -95,23 +105,24 @@ spec:
                 - containerPort: 60000
                   name: metrics
                 resources: {}
+              serviceAccountName: qdrouterd-operator
```

Comment:
Diff showing moved lines, avoid this

Reply:
changes made to fix those unnecessary write spaces and position movement.